### PR TITLE
feat: add chrome extension intro and usage to docs

### PR DESCRIPTION
**Pull request by diivi**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

Fixes https://github.com/open-sauced/browser-extensions/issues/57 and https://github.com/open-sauced/browser-extensions/issues/10 by adding an intro and features listing for the chrome extension.

## Mobile & Desktop Screenshots/Recordings

image

image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [x] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: docs/chrome-extension/introduction-to-the-chrome-extension.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - introduction to the chrome extension
+---
+
+## The OpenSauced Chrome Extension
+
+The OpenSauced Chrome extension seamlessly integrates GitHub with the OpenSauced platform. With this extension, you can easily view and discover open-source projects looking for contributions directly from GitHub, making collaboration and contribution easier than ever.
+
+## Installing the Chrome Extension
+
+To install the OpenSauced Chrome extension, navigate to the [Chrome Web Store](https://example.com) and click the "Add to Chrome" button.
```
Comment (diivi) [inline docs/chrome-extension/introduction-to-the-chrome-extension.md:15]: TBD?

**Comment by bdougie** [comment]
> @bdougie Let me know if you thought of something else. As the extension is under rapid development, the screenshots might need to be updated anyway.

It is live and as far as screen shots this works. We should add the screenshots to the browser extension repo as well. 

Extension is also live. 

https://chrome.google.com/webstore/detail/opensauced/aebjdjjodmkdnmejakflnfpkadblfmdh

File: docs/chrome-extension/introduction-to-the-chrome-extension.md (hunk 1 above)
Comment (bdougie) [reply docs/chrome-extension/introduction-to-the-chrome-extension.md:15]: https://chrome.google.com/webstore/detail/opensauced/aebjdjjodmkdnmejakflnfpkadblfmdh

Comment (bdougie) [inline docs/chrome-extension/introduction-to-the-chrome-extension.md:15]:
```suggestion
To install the OpenSauced Chrome extension, navigate to the [Chrome Web Store](https://chrome.google.com/webstore/detail/opensauced/aebjdjjodmkdnmejakflnfpkadblfmdh) and click the "Add to Chrome" button.
```

**Review by bdougie — APPROVED** [review]

**Pull request merged**